### ISSUE TITLE
Show tooltip when hovering node

### ISSUE THREAD

**Issue by nikolay-borzov** [evmar/webtreemap#28]
Useful when node is too small to fit caption

**Comment by evmar** [comment]
Good idea!  Might be as simple as adding a `title` attribute on the DOM node.

**Comment by nikolay-borzov** [comment]
I believe the first version has it.
But It would be great to provide a hook method where a user can attach custom behaviour to node element (e.g. add multiline tooltip)

**Comment by evmar** [comment]
I guess it would go somewhere around here:

https://github.com/evmar/webtreemap/blob/master/src/treemap.ts#L115

I experimented with letting the user provide a function here, I can't remember why I removed it.
